### PR TITLE
Feat/util pages

### PR DESCRIPTION
**Pull request by Debarshi95**
Pr to add Util Pages and other related components

Deploy: https://62587ed9bd739210231eef29--debarshib-flixnotes.netlify.app/signin
Routes to check: /label, /archive
Creds:
Email:testuser@test.com
Password: 1234567

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Like the way you are handling the error with an error boundary

Like the code structure and quality
good to go.

File: src/components/common/ErrorBoundary/ErrorBoundary.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.errorBoundary__root {
+  max-width: 94%;
+  margin: 0 auto;
```
Comment (dev-ashishsethi) [inline src/components/common/ErrorBoundary/ErrorBoundary.css]: could have used  margin-inline

File: src/components/common/ErrorBoundary/ErrorBoundary.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // eslint-disable-next-line react/no-unused-state
+      error: null,
+      hasError: false,
+    };
+  }
```
Comment (dev-ashishsethi) [inline src/components/common/ErrorBoundary/ErrorBoundary.js:12]: learnt something new thanks!

File: src/components/dashboard/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -27,11 +28,11 @@ const Sidebar = () => {
...
-    {
-      path: '/profile',
-      name: 'Profile',
-      icon: <FaUserCircle className="Sidebar__icon" />,
-    },
+    // {
+    //   path: '/profile',
+    //   name: 'Profile',
+    //   icon: <FaUserCircle className="Sidebar__icon" />,
+    // },
```
Comment (dev-ashishsethi) [inline src/components/dashboard/Sidebar/Sidebar.js]: delete comments

**Review by dev-ashishsethi — APPROVED** [review]
really liked the use of error boundary

File: src/components/dashboard/Sidebar/Sidebar.js (hunk 3 above)
Comment (Debarshi95) [reply src/components/dashboard/Sidebar/Sidebar.js]: Commented part contains link that'll be added after current submission hence added the commented part

**Pull request merged**